pages/docs/how-to-guides/events.mdx

```diff
@@ -53,9 +55,11 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
 | name                   | true     | Name of the event                                           |
 | description            | true     | More details about the event, this can include markdown     |
 | date                   | true     | Start and end date plus time of the event                   |
-| cfpClose               | false    | The date of when the CFP to submit a talk closes            |
 | url                    | true     | Where can people learn more about the event                 |
+| cfpClose               | false    | The date of when the CFP to submit a talk closes            |
 | location               | false    | Address of the event venue                                  |
+| userStatus             | false    | What your role is at the event                                  |
+| speakingTopic          | false    | The topic of the event                                  |
```

Comment:
This is so that it looks ordered with the other elements, shouldn't really affect how it looks in rendered Markdown, though.
```suggestion
| userStatus             | false    | What your role is at the event                              |
| speakingTopic          | false    | The topic of the event                                      |
```